text_extensions_for_pandas/io/watson/test_nlu.py

```diff
@@ -424,8 +424,9 @@ def setUpClass(cls):
     def _make_request():
         from ibm_watson import NaturalLanguageUnderstandingV1
         from ibm_cloud_sdk_core.authenticators import IAMAuthenticator
-        from ibm_watson.natural_language_understanding_v1 import Features, CategoriesOptions, ConceptsOptions, EmotionOptions, EntitiesOptions, KeywordsOptions, \
-            MetadataOptions, RelationsOptions, SemanticRolesOptions, SentimentOptions, SyntaxOptions, SyntaxOptionsTokens
```

Comment:
`MetadataOptions` was invalid, must have been from a very old version.

Reply:
Looks like it has been renamed to from `FeaturesResultsMetadata`, I'll update the comment but this is not currently being used

Reply:
The API reference still lists it as valid, I'll open an issue
https://cloud.ibm.com/apidocs/natural-language-understanding?code=python#metadata

Reply:
Made issue https://github.com/watson-developer-cloud/python-sdk/issues/778